node/core/runtime-api/src/lib.rs

```diff
@@ -276,13 +271,12 @@ where
 	}
 
 	/// Spawn a runtime API request.
-	///
-	/// If there are already [`MAX_PARALLEL_REQUESTS`] requests being executed, the request will be buffered.
 	fn spawn_request(&mut self, relay_parent: Hash, request: Request) {
 		let client = self.client.clone();
 		let metrics = self.metrics.clone();
 		let (sender, receiver) = oneshot::channel();
 
+		// TODO: https://github.com/paritytech/polkadot/issues/5486
```

Comment:
Is it really `TODO`? It is not clear from the code.

Reply:
Yeah, probably I did not really make it clear. The issue documents the problem of using unbounded internal channels and the not so efficient caching.  This PR just handles the first part. I should probably create a new issue for the cache and mention it here.

Reply:
https://github.com/paritytech/polkadot/issues/5546